public/app/plugins/datasource/prometheus/datasource.tsx

```diff
@@ -802,10 +798,12 @@ export class PrometheusDatasource
 
   getExemplars(query: PromQueryRequest) {
     const url = '/api/v1/query_exemplars';
```

Comment:
same question as above. should this be a standard query instead of a resource call?

Reply:
Correct. Look a little further down, `_request` is used, rather than `metadataRequest` which uses the resource calls